.codespellrc

```diff
@@ -0,0 +1,7 @@
+[codespell]
+# skip most common present files which might be on the system
+skip = venvs,.venv,.git,build,*.egg-info,*.lock
+# Flage and Constans are the names
```

Comment:
"the names" implies a particular context for what the names are of. Either remove "the" or add more information about why these names are significant.

```suggestion
# Flage and Constans are names
```